ddtrace/internal/sampling.py

```diff
@@ -112,18 +112,16 @@ class SpanSamplingRule:
 
     def __init__(
         self,
+        sample_rate,  # type: float
+        max_per_second,  # type: int
         service=None,  # type: Optional[str]
         name=None,  # type: Optional[str]
```

Comment:
Is there a case where `SpanSamplingRule` is initialized without the `service` and `name` arguments? Should we remove the default values?

Reply:
Yeah neither of service nor name are required, although there always has to be at least one.